values.yaml

```
@@ -26,6 +26,15 @@ bootstrap:
   initialAdminUser:
     password:
       value: "admin"
+  db:
+    cluster:
+      password:
+        valueFrom:
+          secretKeyRef:
+            name: "prereqs-postgresql"
+            key: "password"
+      username:
+        value: "user"
```

No reason why this should be different from what we have in the CI.
```suggestion
        value: "postgres"
```